Sensitivity.jmd

```diff
@@ -0,0 +1,135 @@
+---
+weave_options:
+  error: false
+  echo: false
+---
+
+```julia
+import Pkg; Pkg.instantiate()
+
+Pkg.activate(".")
+using DrugResponseModel
+using ExponentialUtilities
+using Plots
+Plots.scalefontsizes(0.7)
+```
+To show the sensitivity of cell number difference predicted by combination.
+
+```julia
+concs, populations1, g1s1, g2s1 = load(189, 1);
+concs, populations2, g1s2, g2s2 = load(189, 2);
+concs, populations3, g1s3, g2s3 = load(189, 3);
+g0 = g1s1[1,1,1]+g2s1[1,1,1];
+
+g1avg = (g1s1 .+ g1s2 .+ g1s3) ./ 3; # pure data
+g2avg = (g2s1 .+ g2s2 .+ g2s3) ./ 3; # pure data
+popavg = g1avg .+ g2avg;
+IC50_lap, IC50_dox, IC50_gem, IC50_tax, IC50_pal = DrugResponseModel.find_IC(popavg, 0.5);
+ic50s = [IC50_lap, IC50_dox, IC50_gem, IC50_tax, IC50_pal]
+```
+
+```julia
+p = [97.7434, 1.41726, 0.0501761, 0.930205, 0.111494, 0.123604, 0.552216, 
+        92.9045, 1.40024, 0.996396, 0.0513087, 0.521831, 0.6535, 0.566578, 
+        15.5317, 2.3689, 0.592433, 0.999986, 0.0283363, 0.286975, 0.503328, 
+        3.96929, 4.62768, 0.0512281, 0.307528, 0.549714, 0.378717, 0.50959, 
+        63.4248, 0.976052, 0.16582, 0.740009, 0.0572609, 0.0776912, 0.534201, 
+        0.734513, 0.375555, 16.8387, 12.3945, 30.176, 14.5352]
+    g0 = 20.0
+
+function decide_index(Dr1Ind)
+    if Dr1Ind == 1
+        indexes1 = 1:6
```

Comment:
If `p` is a 2D array you don't need this complicated indexing.

Reply:
I'll make a function to create the 2D from the `p` in the new branch.